__tests__/components/editor/ResourceTemplateForm.test.js

```diff
@@ -112,6 +112,7 @@ describe('<ResourceTemplateForm /> after fetching data from sinopia server', ()
   describe('configured component types', () => {
     it('renders a lookup component', async () => {
       const rtProps = {
+        rtId: 'resource:schema:Name',
```

Comment:
If this is meant to be an actual resourceTemplateId, a better fake value would be 'myOrg:rt:myTemplate' - that is closer to what we will be seeing, and may make this less confusing to read in future.